executor/executor_bsd.h

```diff
@@ -35,26 +35,10 @@ static long execute_syscall(const call_t* c, long a[kMaxArgs])
 	return __syscall(c->sys_nr, a[0], a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8]);
 }
 
-#if GOOS_freebsd
-
-#define KIOENABLE _IOW('c', 2, int) // Enable coverage recording
```

Comment:
I am not sure what's the state for FreeBSD. The last time the KCOV patch still wasn't merged. @tuexen, can we replace this with `#include <sys/kcov.h>`?

Reply:
It is not yet merged (see [D14599](https://reviews.freebsd.org/D14599). Eventually, we should replace the defines with `#include <sys/kcov.h>` and need some trivial changes, but right now this doesn't work yet. Once it is included in FreeBSD head, I can submit a patch to use the include file and to fix the usage of the sysctl's.

Reply:
Then I think we need to revert FreeBSD part for now.
Anton, please also add the https://reviews.freebsd.org/D14599 link on top of FreeBSD declarations while you are here. Thanks.